Src/IronPython/Modules/array.cs

```diff
@@ -28,6 +28,8 @@ public static class ArrayModule {
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2104:DoNotDeclareReadOnlyMutableReferenceTypes")]
         public static readonly PythonType/*!*/ ArrayType = DynamicHelpers.GetPythonTypeFromType(typeof(array));
 
+        public static readonly string typecodes = "cbBuHhiIlLfd";
```

Comment:
Python 3 doesn't include the `c` typecode. Did you include it because we're currently implementing it?

Reply:
Yes; the string was taken directly from the list of characters in `CreateData()`.

On a related note, we'll eventually need to implement `q` and `Q`.